core/auth/interfaces/controller.go

```diff
@@ -0,0 +1,48 @@
+package interfaces
+
+import (
+	"context"
+	"errors"
+	"net/url"
+	"flamingo.me/flamingo/v3/framework/web"
+)
+
+type (
+	Authcontroller struct {
+		responder *web.Responder
+		authservice Authservice
+		router         web.ReverseRouter
+	}
+)
+
+
+// Inject for Authcontroller
+func (c *Authcontroller) Inject(responder *web.Responder, authservice Authservice,router web.ReverseRouter,) {
+	c.responder = responder
+	c.authservice = authservice
+	c.router = router
+}
+
+//AuthAction - default auth action - starting the authorsation with the registered Authservice
+func (c *Authcontroller) AuthAction(ctx context.Context, r *web.Request) web.Result {
+	redirecturl, ok := r.Params["redirecturl"]
```

Comment:
You can safely omit the `ok` variable here, the default value for string is `""`